src/web/lib/contributions.ts

```diff
@@ -127,19 +127,19 @@ export const shouldHideSupportMessaging = (
 
 const REQUIRED_CONSENTS_FOR_ARTICLE_COUNT = [1, 3, 7];
 
-export const hasOptedOutOfArticleCount = (): boolean =>
+export const hasArticleCountOptOutCookie = (): boolean =>
 	getCookie(OPT_OUT_OF_ARTICLE_COUNT_COOKIE) !== null;
 
 const removeArticleCountsFromLocalStorage = () => {
 	window.localStorage.removeItem(DAILY_ARTICLE_COUNT_KEY);
 	window.localStorage.removeItem(WEEKLY_ARTICLE_COUNT_KEY);
 };
 
-export const getArticleCountConsent = (): Promise<boolean> => {
-	if (hasOptedOutOfArticleCount()) {
-		return Promise.resolve(false);
-	}
+export const hasCmpConsentForArticleCount = (): Promise<boolean> => {
 	return new Promise((resolve) => {
+		if (getCookie('gu-cmp-disabled')) {
+			resolve(true);
```

Comment:
this prevents the cypress tests from breaking, as `onConsentChange` doesn't work